08_files_lab/02.py

```diff
@@ -1,26 +1,10 @@
-"""
-Write a program that takes 3 names. The first two
-are existing file names and the last is a new file name/
-The program should write the lines from the first two 
-files interwinded into the output file. So if we have
-
-a.txt with
-a1
-a2
-
-and b.txt with
-b1
-b2
-
-then running
-
-python 02.py a.txt b.txt c.txt
-will result in the creation of c.txt with the following content:
-
-a1
-b1
-a2
-b2
-
-"""
-
+"""intsert 2 file to one, line by line"""
+import os,sys
+from itertools import izip_longest
+f1 = sys.argv[1]
+f2 = sys.argv[2]
+f3 = sys.argv[3]
+with open ('c:\python/week_2_2/{0}'.format(f1),'r') as fin1,open ('c:\python/week_2_2/{0}'.format(f2),'r') as fin2,open ('c:\python/week_2_2/{0}'.format(f3),'w') as fout :

```

Comment:
I assume you ran this program from a different directory and wanted to load a file name relative to the directory of the script (rather than current working directory the script was executed at).

i.e. I assume your script was in `c:\python\week_2_2` but you called it from `C:\` or something specifying full path to the script.

In this case you can use `__file__` to get the current python script file name, and with os module get its path:

```
import os
print(os.path.dirname(os.path.realpath(__file__)))
```